ts/Core/Axis/ColorAxis.ts

```diff
@@ -1579,7 +1579,7 @@ namespace ColorAxis {
         chart: Chart;
         name: string;
         options: object;
-        drawLegendSymbol: LegendSymbolMixin['drawRectangle'];
+        drawLegendSymbol: typeof LegendSymbolMixin['drawRectangle'];
```

Comment:
Why is this reduced to a function type?

Reply:
It appeared as a problem in the editor. I got the message 'LegendSymbolMixin' refers to a value, but is being used as a type here. Did you mean 'typeof LegendSymbolMixin'?

Reply:
Okay, thanks.